### ISSUE TITLE
Does it support Air Purifier 3C?

### ISSUE THREAD

**Issue by malditavibora** [rgavril/homebridge-xiaomi-air-purifier3#24]
Cannot make it work with model zhimi.airpurifier.mb4 (Air Purifier 3C) the icon appears in Home app but as No response.

**Comment by kaiserkiwi** [comment]
Same problem here. Got my 3C yesterday and it shows up in my Home app but sadly no response.

**Comment by samanoju** [comment]
yes, same here. none of plugins available currently supports 3C model.

**Comment by macgyverkam** [comment]
Try this: https://github.com/merdok/homebridge-miot

**Comment by fsrxc** [comment]
> Try this: https://github.com/merdok/homebridge-miot

works great with my new 3C...thank you

**Comment by Stetocasa** [comment]
Guys, the alternative plugin (miot) is 10 times bad compare this one (that not works with 3C)
In Miot the device is exposed as a normal switch and no possibility to obtain temperature, humidity and air quality.
@rgavril any chance to let your plugin works with 3C Model?
Thanks

**Comment by kaiserkiwi** [comment]
The Miot-Plugin has the ability to expose these properties. My 3C shows me the air quality level just fine. With the `pm25Breakpoints` option I can even configure the breakpoints.

If you have any problems with the plugin, the support is really good (Not like this one here…). The dev helped me alot.
